### PR TITLE
Project 3 - Isabella Billgren

### PR DESCRIPTION
**Pull request by Bellsibub**
Completed all BLUE, RED and BLACK requirements.
(see readme for more info)

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -4,27 +4,274 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="main.css" />
```
Comment (EmOlsz) [inline code/index.html:7]: Next time let's remember about `./` in path to file ;)

File: code/index.html [hunk 2]
```diff
@@ -4,27 +4,274 @@
...
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- Header with full width video & nav links to login -->
+    <header class="hero">
+      <video class="hero__video" playsinline autoplay muted loop>
```
Comment (EmOlsz) [inline code/index.html:12]: Great job with BEM methodology!

File: code/index.html [hunk 3]
```diff
@@ -4,27 +4,274 @@
...
+        <source src="./assets/videos/adventures.mp4" type="video/mp4" />
+        <source src="./assets/videos/adventures.webm" type="video/webm" />
+        Sadly your browser not support HTML 5 video.
+      </video>
+      <h1 class="hero__title">
+        Let the <br />
```
Comment (EmOlsz) [inline code/index.html:18]: We shouldn't use `<br />` tags - more information on that in [Week 2 Master document](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/index.html [hunk 4]
```diff
@@ -4,27 +4,274 @@
...
+      <div class="content__container">
+        <h2 class="content__title">Book your adventure!<span>Learn the wilderness</span></h2>
+        <!-- The Form -->
+        <form class="content" action="https://httpbin.org/anything" method="post">
+          <!-- Form part 1 -->
+          <!-- Contact information -->
+          <fieldset class="form__section">
+            <legend class="form__section-title">Who is booking?</legend>
+            <!-- Name (text) -->
+            <label class="form__input-wrapper" for="name">
```
Comment (EmOlsz) [inline code/index.html:38]: If we decide to nest input inside label, then we don't need to add for - id pair. More details in Week 3 Master document

File: code/index.html [hunk 5]
```diff
@@ -4,27 +4,274 @@
...
+          </div>
+        </form>
       </div>
+    </section>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <!-- Section containing the accordion -->
+    <section class="accordion">
```
Comment (EmOlsz) [inline code/index.html:227]: Great accordion structure :)

File: code/script.js [hunk 6]
```diff
@@ -1,9 +1,45 @@
...
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+const accordion = document.getElementById("accordion");
+
+const plusIcon = "&#10133;",
+  minusIcon = "&#10134";
+
+let currentQuestion;
+
+accordion.addEventListener("click", (event) => {
```
Comment (EmOlsz) [inline code/script.js:8]: Very complex logic, I'm impressed!

File: code/styles/forms.css [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+.form__input-radio {
+  /* make the radio button round */
+  border-radius: 10px;
+}
+
+/* -CHECKBOX&RADIO(states)-  */
+
+/* Checked style of checkbox and radio */
+.form__input-checkbox:checked::before,
+.form__input-radio:checked::before {
```
Comment (EmOlsz) [inline code/styles/forms.css:83]: 👍

File: code/styles/forms.css [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+  cursor: default;
+}
+
+.form__input-field--disabled:hover {
+  background: #f3f3f3;
+}
+
+/* fix to make sure that the labels are aligned with the custom Checkboxes and radios */
+label,
+label.form__input-wrapper--submit-section {
```
Comment (EmOlsz) [inline code/styles/forms.css:119]: Would be better to add separate class for `label`, let's remember about this next time!

File: code/styles/gridLayout.css [hunk 9]
```diff
@@ -0,0 +1,48 @@
+/* 
+#8d8741;
+#659dbd;
+#daad86;
+#bc986a;
+#fbeec1; 
+*/
```
Comment (EmOlsz) [inline code/styles/gridLayout.css:7]: All commented code should be removed before final push - let's fix this

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Wow Isabella, congrats! ⭐️ Very nice, well done project. I'm impressed by your BEM implementation, custom input styles and JS logic. There are only super small things that needs to be fixed before approval, check them out :)
Moreover, feel free to analyse [Week 3 Master document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475)

**Comment by Bellsibub** [comment]
Requested changes have been pushed :)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**